### DISCUSSION TITLE
Don't hide background after [end_timeline]

### DISCUSSION THREAD

**Discussion by McStebb** [dialogic-godot/dialogic#1813] [Q&A] [answered]
Hi there,

I'm guessing there's a simple way to do this and I just haven't found it yet.
In our VN project I sometimes pull up a menu overlay to enable navigation.

I want the dialog to stop playing while this happens, so I put the navigation menu between 2 timeline using `[end_timeline]`, and then whichever button I click on the navigation menu will start the next timeline with `Dialogic.start()`

My problem is that `[end_timeline]` also hides the background. Is there a good way to stop timeline playback without removing the background, so whichever background was last displayed stays on-screen until a new one is called in to replace it?

Thank you

**Comment by Jowan-Spooner** [comment, marked as answer]
Hey, 
by default the layout will only be kept while dialog is running and discarded afterwards. You can change this behaviour in the dialogic settings > general.
1. Set the `layout node behaviour` to "Keep Layout Node" 
2. Then use a `clear` event with "clear background" disabled (in text: `[clear background="false"]`) before the [end timeline].

However while testing this I noticed [clear] won't hide the textbox (which it should, probably with a setting). I will make a bug report. For now the only proper way to hide the textbox is to use a wait event with a very short time and hide_textbox on.

**Reply by McStebb** [reply]
This works great for now! Thank you!
